lib/utils/ethers-execute.ts

```diff
@@ -0,0 +1,62 @@
+import { ethers } from 'ethers'
+import { TxReceipt } from './web3-txs'
+
+type Option = {
+	readonly contract: ethers.Contract
+	readonly method: string
+	readonly args?: ReadonlyArray<string | boolean>
+	readonly mutation?: boolean
+}
+
+export type QueryOption = Option & {
+	readonly mutation: false
+}
+
+export type MutationOption = Option & {
+	readonly mutation: true
+}
+
+export type MutationReturn = {
+	readonly wait: () => Promise<TxReceipt>
```

Comment:
In Ethers, can be use the type `TransactionResponse` instead of `TxReceipt`, and the document here: https://docs.ethers.io/v5/api/providers/types/#providers-TransactionResponse

Reply:
Sorry, I made misunderstand...

Reply:
The type `MutationReturn` should be replaced by `TransactionResponse`. TransactionResponse includes `.wait` as well.

Reply:
oh my god!

okok